README.md

```diff
@@ -283,7 +284,7 @@ player.record().destroy();
 | Method | Description |
 | --- | --- |
 | `isRecording` | Returns a boolean indicating whether recording is active or not. |
-| `getRecordType` | Get recorder type as string. Either `image_only`, `animation`, `screen_only`, `audio_only`, `video_only` or `audio_video`. |
+| `getRecordType` | Get recorder type as string. Either `image_only`, `animation`, `screen_only`, `audio_only`, `video_only`, `AUDIO_SCREEN` Or `audio_video`. |
 | `saveAs` | Show save as dialog in browser so the user can [store the recorded media locally](#save-data). |
```

Comment:
use the constant value here, `audio_screen`

Reply:
Fixed in rebase 0200355